sdk/core/src/services/WalletStorageService.ts

```diff
@@ -161,6 +164,12 @@ export default class WalletStorageService {
   }
 
   async authorizeVcVault() {
+    const headers: any = {}
```

Comment:
@rbrazhnyk such header should be sent to each request to the bloom vault, not only auth